draft-ietf-tls-subcerts.md

```diff
@@ -519,24 +533,23 @@ Cloudflare's IANA Private Enterprise Number (PEN) arc.
 
 # Security Considerations
 
-## Security of delegated private key
+## Security of delegated credential's private key
 
-Delegated credentials limit the exposure of the TLS private key by limiting
-its validity.  An attacker who compromises the private key of a delegated
-credential can act as a man-in-the-middle until the delegate credential expires,
-however they cannot create new delegated credentials.  Thus, delegated
+Delegated credentials limit the exposure of the private key used in a TLS connection by limiting
+its validity period.  An attacker who compromises the private key of a delegated
+credential can act as a man-in-the-middle until the delegated credential expires.
+However, they cannot create new delegated credentials.  Thus, delegated
 credentials should not be used to send a delegation to an untrusted party, but
 is meant to be used between parties that have some trust relationship with each
-other.  The secrecy of the delegated private key is thus important and several
+other.  The secrecy of the delegated credential's private key is thus important and several
 access control mechanisms SHOULD be used to protect it, including file system
 controls, physical security, or hardware security modules.
 
 
 ## Re-use of delegated credentials in multiple contexts
 
-It is possible to use the same delegated credential for both client and server
-authentication if the Certificate allows it.  This is safe because the context
-string used for delegated credentials is distinct in both contexts.
+It is not possible to use the same delegated credential for both client and server
+authentication because the signature contains a context string.
```

Comment:
```suggestion
authentication because endpoints compute the corresponding signature using a context string unique to their role (client or server).
```

Reply:
It's about the signature *of* the DC, which contains a context string to determine which peer can use it.

Reply:
Right, it's this signature:

~~~
   struct {
     Credential cred;
     SignatureScheme algorithm;
     opaque signature<0..2^16-1>;
   } DelegatedCredential;
~~~

Which is computed using a string with either "client" or "server" in it. Let me re-phrase the suggestion!